src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
```

Comment:
You should place each value of attributes (with multiple values) in new line.

 See more details in screenshot 
![image](https://user-images.githubusercontent.com/111275759/193404034-b95e4afe-addb-4220-b22f-7456d2aa0ac3.png)

Reply:
this tag has less than 3 attributes. but you are right it has more than 80 symbols in a row.

Reply:
```suggestion
          content="
            width=device-width, 
            user-scalable=no, 
            initial-scale=1.0, 
            maximum-scale=1.0, 
            minimum-scale=1.0
            "
          >
```